test/integration/resize_test.go

```diff
@@ -49,20 +50,20 @@ var _ = Describe("vary VM parameters: memory cpus, disk", Label("openshift-prese
 		})
 
 		It("check VM's memory size", func() {
-			out, err := SendCommandToVM("cat /proc/meminfo")
+			out, err := SendCommandToVM("cat /proc/meminfo", preset.OpenShift)
```

Comment:
`SendCommandToVM` does not really need a preset arg as it's hardcoded to `OpenShift`. In my opinion, it would make more sense to add a constant equal to `preset.OpenShift` and use this instead. OpenShift will be hardcoded in a single place, and won't be hardcoded in SendCommandToVM, which I think is what you want to avoid.

Reply:
`SendCommandToVM` is generic, it needs the preset to call `constants.InstanceName()` to get the VM name, and get the connection details for it, it can be used for the microshift preset with `SendCommandToVM("cat /proc/meminfo", preset.MicroShift)`.

i am not sure if it'd make more sense to use something like:
```
preset := preset.OpenShift
out, err := SendCommandToVM("cat /proc/meminfo", preset)
```
or did i completely misunderstood the comment?

Reply:
i see since this preset is being used in all the invocation, it does make sense to use the above ^^